libfwupdplugin/fu-fuzzer-firmware.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "@INCLUDE@"
+
+int
+LLVMFuzzerTestOneInput(const guint8 *data, gsize size)
+{
+	g_autoptr(FuFirmware) firmware = FU_FIRMWARE(@FIRMWARENEW @());
```

Comment:
gah, it looks like this breaks though from style with the space :(

Reply:
I guess the right answer is to just fixup the file manually and leave it out of pre-commit then ok.

Reply:
I pushed just that one fixed-up hunk to main. Thanks.